src/api/app/components/watchlist_icon_component.html.haml

```diff
@@ -8,9 +8,12 @@
                     action: toggle_watchable_path }) do
     %i.fas.fa-eye
     Unwatch
+    = watch_count_tag
```

Comment:
Let's render in the view, it's a bit more concise even if we repeat ourselves
```suggestion
    - unless watch_count == 0
      %span.badge.bg-secondary.rounded-pill= watch_count
```

Reply:
I still need a check for `foster_collaboration` beta feature. Having another check in the view, repeated twice is not nice. Would be better to leave this piece in the component.

Reply:
We established during planning this doesn't require a beta participation :)

Reply:
Oh okay. I did not know that. :slightly_smiling_face: